File: app/src/main/java/life/plank/juna/zone/data/network/model/FeedItemComment.java
package life.plank.juna.zone.data.network.model;

import java.util.List;

import lombok.Data;

@Data
public class FeedItemComment {
    private String id;
    private String message;
    private String commenterDisplayName;
    private String commenterProfilePicUrl;
    private long likeCount;
    private boolean hasLiked;
    private long replyCount;
    private List<FeedItemCommentReply> replies;
}

File: app/src/main/java/life/plank/juna/zone/data/model/FeedItemComment.kt
package life.plank.juna.zone.data.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize
import kotlinx.android.parcel.RawValue
import java.util.*

@Parcelize
data class FeedItemComment(
        var id: String,
        var message: String,
        var commenterDisplayName: String,
        var commenterProfilePictureUrl: String?,
        var likeCount: Long,
        var hasLiked: Boolean,
        var replyCount: Long,
        var time: Date,
        var replies: @RawValue List<FeedItemCommentReply>?
) : Parcelable